paasta_tools/cli/cmds/spark_run.py

```diff
@@ -808,7 +841,17 @@ def paasta_spark_run(args):
         return 1
 
     volumes = instance_config.get_volumes(system_paasta_config.get_volumes())
-    app_base_name = get_spark_app_name(args.cmd or instance_config.get_cmd())
+    app_base_name = get_spark_app_name(args.cmd or instance_config.get_cmd(), args.autogen)
+
+    if args.autogen != '-1':
+        pod_template_path = "/nail/tmp/podTemplate.yaml"
+        if args.spark_args:
+            args.spark_args += f" spark.kubernetes.executor.podTemplateFile={pod_template_path}"
+            args.spark_args += f" spark.kubernetes.allocation.batch.size=2"
+        else:
+            args.spark_args = f"spark.kubernetes.executor.podTemplateFile={pod_template_path}"
+            args.spark_args += f" spark.kubernetes.allocation.batch.size=2"
```

Comment:
Reason for setting this to 2? 
Any values set to constants need to defined as constant at the top. 
This helps in easier debugging and understanding the code, to go through all the constants from the top.

Reply:
smaller batch size tend to make pod affinity more pronounced

Reply:
Any downsides of keeping the batch size this low (2)? (Example on performance)